test/integration/scheduled-action-integration.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { expect } from 'chai'
+import { before, describe, test } from 'mocha'
+import { Space } from '../../lib/entities/space'
+import { TestDefaults } from '../defaults'
+import { getDefaultSpace, getPlainClient } from '../helpers'
+import { makeLink } from '../utils'
+
+describe('Scheduled actions api', async function () {
+  let testSpace: Space
+
+  before(async () => {
+    testSpace = await getDefaultSpace()
+  })
+
+  describe('Read', () => {
+    test('Get Scheduled action', async () => {
+      const createdAction = await testSpace.createScheduledAction({
+        entity: makeLink('Entry', TestDefaults.entry.testEntryId),
+        action: 'publish',
+        scheduledFor: {
+          datetime: new Date().toISOString(),
```

Comment:
Heads up that you might want to add a few minutes/hours to this or you'll see flakiness due to time issues